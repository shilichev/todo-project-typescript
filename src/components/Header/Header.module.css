  .header {
    width: 1250px;
    height: 80px;
    margin: auto;
    padding-bottom: 50px;
    display: flex;
    justify-content: space-between;
    align-items: center;
  }
  .projectName {
    margin-left: 0;
    margin-right: 30px;
    font-size: 40px;
  }
  .name {
  
    margin-left: 30px;
    margin-right: 0;
    padding: 0;
    font-size: 40px;
  }
  