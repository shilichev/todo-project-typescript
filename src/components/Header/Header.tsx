import classes from "./Header.module.css";
import React from "react";
const Header = () => (
  <div className={classes.header}>
    <p className={classes.projectName}>Project ToDo</p>
    <p className={classes.name}>Shilichev Daniil</p>
  </div>
);

export default Header;
