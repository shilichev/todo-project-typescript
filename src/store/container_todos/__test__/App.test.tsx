import { shallow } from "enzyme";
import React from "react";
import { toJson } from "enzyme-to-json";
import { render, screen } from '@testing-library/react';
import Header from "../../../components/Header/Header";

it('renders learn react link', () => {
  // render(<Header />);
  // const linkElement = screen.getByText(/Project ToDo/i);
  // expect(linkElement).toMatchSnapshot();
  const wrapper = shallow(<Header />);
  expect(toJson(wrapper)).toMatchSnapshot();
});