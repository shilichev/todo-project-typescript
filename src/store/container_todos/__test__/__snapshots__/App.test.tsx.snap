// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<p>
  Project ToDo
</p>
`;
